composer.json

```diff
@@ -25,10 +25,5 @@
         "psr-0": {
             "Webfactory": "src/"
         }
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^8.5",
-        "satooshi/php-coveralls": "^0.6.1",
-        "kherge/box": "^2.6"
```

Comment:
The Travis CI build is failed because the `require-dev` block has been deleted.

And the `composer` will not install development dependencies during `composer install` work.

Reply:
And the `phpunit` and `php-coveralls` execution PHP scripts will not on `vendor/bin` folder.